app/GraphQL/Ecosystem/Mutations/Auth/Auth.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace App\GraphQL\Ecosystem\Mutations\Auth;
+
+use Exception;
+use Illuminate\Support\Facades\Auth as AuthFacade;
+use  Kanvas\Users\Repositories\UsersRepository;
+
+class Auth
+{
+    /**
+     * changePassword.
+     *
+     * @param  mixed $root
+     * @param  array $req
+     *
+     * @return void
+     */
+    public function changePassword(mixed $root, array $req)
+    {
+        if ($req['new_password'] == $req['new_password_confirmation']) {
```

Comment:
@FredPeal use laravel validation for this part
send a user notification once pass is update
use user make password method on the user model